frontend/src/components/auth/InputForm.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <div>
+    <slot></slot>
+  </div>
+</template>
+
+<script lang="ts">
+export default {};
+</script>
+
```

Comment:
Let's merge the image component to this component, and rename it to AuthComponent

Reply:
I import image component in login view, and this component in login card, how can I merge them?